src/Microsoft.Graph.Cli.Core/IO/GraphCliClientFactory.cs

```diff
@@ -22,6 +20,31 @@ public static HttpClient GetDefaultClient(GraphClientOptions? options = null, st
 
         m.AddRange(middlewares);
 
+        // Add logging handler.
+        if (loggingHandler is LoggingHandler lh)
+        {
+            m.Add(lh);
+        }
+
+        // Set compression handler to be last (Allows logging handler to log request body)
```

Comment:
doesn't the http message handler need to be last in the case of dotnet?

Reply:
It's always the last because it's set as the `finalHandler`

https://github.com/microsoftgraph/msgraph-cli-core/blob/b29389e03c5df3905f926bfeabfd99954a25b639/src/Microsoft.Graph.Cli.Core/IO/GraphCliClientFactory.cs#L11

https://github.com/microsoftgraph/msgraph-cli-core/blob/b29389e03c5df3905f926bfeabfd99954a25b639/src/Microsoft.Graph.Cli.Core/IO/GraphCliClientFactory.cs#L48